### ISSUE TITLE
Can you make this into a Userscript please?

### ISSUE THREAD

**Issue by thinkpad4** [cryptodescriptor/old-twitch#1]
Thank you very much

**Comment by cryptodescriptor** [comment]
By userscript do you mean greasemonkey for example? Do you not have Ublock or Adblock+ (I think they use the same formatting).

**Comment by thinkpad4** [comment]
Yeah, Greasemonkey, Violentmonkey.  I suggest userscript cause if/when Twitch makes more changes to their code it would make updating your code easier, instead of having to copy and paste the new code.

**Comment by cryptodescriptor** [comment]
I have decided to use Stylus plugin which is available for Chrome, Firefox, and Opera. It is less intrusive than Greasemonkey, and you don't need a userscript manager like Violentmonkey. Stylus was designed strictly for editing a websites styles, so I feel like this is a better choice for old-twitch. Automatic updates occur at a set interval, I have updated README.md to reflect these new changes. Firefox and Chrome standalone plugins will still be available for now, but if I have to keep updating them I might just drop support.
